### PR TITLE
Re former

### PR DESCRIPTION
**Pull request by Bishoy-Samwel**
- Creating new user
- Edit Existing users
- There are validations for user inputs

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!--
+
+<form action="#create" method="get" accept-charset="UTF-8">
```
Comment (ElijahTobs) [inline app/views/users/new.html.erb]: - [x] It should use `users_path` helper for action

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+<input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="user[username]"><br><br>
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br><br>
```
Comment (ElijahTobs) [inline app/views/users/new.html.erb:10]: - [x] You're expected to use `form_authenticity_token`

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  <input type="submit" value="Submit">
+</form> 
+
+-->
+<!--
+<%= form_tag("#create", method: "get") do %>
```
Comment (ElijahTobs) [inline app/views/users/new.html.erb]: - [x] Your `form_tag` should also use `user's path` for helper action
- [x] It should also use `label_tag`, `text_field_tag`, `email_field_tag`, `password_field_tag` like so:

```html
<%= form_tag(users_path) do %>
  <%= label_tag 'username' %> <%= text_field_tag 'username' %>
  <%= label_tag 'email' %> <%= email_field_tag 'email' %>
  <%=label_tag 'password' %> <%= password_field_tag 'password' %> <%= submit_tag %>
<% end %>
```

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @Bishoy-Samwel and @CollinsTatang  :wave: 

You've done well so far :+1:  and I commend you for that :clap:
However, there are still some changes to be made before this milestone can be approved.

## REQUIRED

- Check inline comments below




### Make the requested changes and submit for another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED✔✔**

Great work making the required changes.👍👍
This is some great work.👏👏👏

You can merge👌👌
Happy coding💻💻💻💻

**Pull request merged**